### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by uncenter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,88 @@
...
-{
-  "name": "Example Website",
-  "slug": "example",
-  "about": "I'm an example!!!",
-  "url": "https://example.com/",
-  "rss": "https://example.com/index.xml",
-  "owner": "person@example.com"
-}
-```
+1. Go to...
```
Comment (isabelroses) [inline README.md:39]: Remove this please

Comment (uncenter) [reply README.md:39]: Remove what? The placeholder? The website wasn't working yet soo....

Comment (uncenter) [reply README.md:39]: Oh all the previews? We should have previews at some point no?

Comment (isabelroses) [reply README.md:39]: I mean, it feels weird to me.

**Pull request merged**